src/components/views/ViewMilestone.jsx

```diff
@@ -171,6 +178,41 @@ class ViewMilestone extends Component {
                 </div>
               </div>
 
+              { items && items.length > 0 &&
+                <div className="row spacer-top-50 dashboard-table-view">
+                  <div className="col-md-8 m-auto">
+                    <h4>Milestone items</h4>
+
+                    {/* MilesteneItem needs to be wrapped in a form or it won't mount */}
+                    <Form>
+                      <div className="table-container">
+                        <table className="table table-responsive table-striped table-hover">
+                          <thead>
+                            <tr>
+                              <th className="td-item-date">Date</th>                        
+                              <th className="td-item-description">Description</th>
+                              <th className="td-item-amount-fiat">Amount Fiat</th>
+                              <th className="td-item-fiat-amount">Amount Ether</th>
```

Comment:
This is confusing. You have class names `td-item-amount-fiat` and `td-item-fiat-amount`. Is this a typo?